packages/python-backoff/python-backoff.spec

```diff
@@ -3,10 +3,10 @@
 
 Name:           python-%{pypi_name}
 Version:        1.10.0
-Release:        2%{?dist}
+Release:        3%{?dist}
 Summary:        Function decoration for backoff and retry
 
-License:        None
```

Comment:
This one is weird because you'd think https://github.com/litl/backoff/blob/128bba5377568b8e2c0230973e887e0b4f5da800/pyproject.toml#L8 would be sufficient